test/help.test.js

```diff
@@ -45,6 +46,26 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  describe('prints right help message for each generator', function() {
+    var cmdNames = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
+    'middleware', 'model', 'property', 'relation', 'remote-method', 'swagger'];
+
+    cmdNames.forEach(function(name) {
+      it('prints right help message for generator ' + name, function() {
+        process.env.SLC_COMMAND = 'slc loopback --help';
```

Comment:
See my older comment, I think this should use only `slc` as the command:

```js
process.env.SLC_COMMAND = 'slc';
```

Unless I am missing something?